Template.yaml

```diff
@@ -0,0 +1,648 @@
+apiVersion: v1
```

Comment:
Shall we add the license header in this file

Reply:
Resolve with the commit 70554b4